src/cli.rs

```diff
@@ -525,6 +533,21 @@ pub fn handle_cli_asset_command(command: &AssetCommands) {
                 }
             }
         }
+        AssetCommands::BearerAuth { uuid, token } => {
+            let asset_command = commands::asset::AssetCommand::new(authentication);
+            match asset_command.update_web_asset_headers(
+                uuid,
+                vec![("Authorization".to_owned(), format!("Bearer {token}"))],
```

Comment:
Bearer is used for jwt authentication - we don't use it much though and don't expose it yet anywhere.

Reply:
@korvyashka I don't understand your point - this is for customers to use. To set it for their web assets.

Reply:
This was my request, @korvyashka. We need it for a number of the tutorials.